### DISCUSSION TITLE
Github Org logo

### DISCUSSION THREAD

**Discussion by sdboyer** [pollypkg/polly#5] [Ideas]
We need an org logo for polly packages. (Branding FTW!)

My personal thoughts run to some kind of vectorized parrot - like, half a picture of a parrot (Polly), and half the parrot reduced to the points on the path of a vector diagram defining the shape of the parrot (poly, since what we have is a little mathy).

Or something totally different, having to do with Obs and packages. 🤷 

/begin bikeshed

**Comment by tombrk** [comment]
![fat parrot](https://dcassetcdn.com/w1k/submissions/348000/348069_2acd.jpg)

**Reply by metalmatze** [reply]
<img src="https://cultofthepartyparrot.com/parrots/hd/parrot.gif" width="25"><img src="https://cultofthepartyparrot.com/parrots/hd/parrot.gif" width="25"><img src="https://cultofthepartyparrot.com/parrots/hd/parrot.gif" width="25">

**Comment by sdboyer** [comment]
OMG OMG HOW DID I MISS POLLY POCKET

![polly pocket](https://media.giphy.com/media/FCJ1gFWXLBttu/source.gif)

**Comment by RichiH** [comment]
I immediately thought of:
* Parrot
* pulling at something
* POLYgons
* poly colours
* the letter P

**Reply by RichiH** [reply]
Well, and https://cultofthepartyparrot.com/

**Reply by sdboyer** [reply]
i love it when you find out you're in a cult and didn't even know it
